spec/services/void_participant_declaration_spec.rb

```diff
@@ -62,9 +62,12 @@
         )
       end
 
-      it "can be voided" do
+      let(:line_item) { participant_declaration.statement_line_items.first }
```

Comment:
Maybe we group the specs correctly, as the below context for `when declaration is attached to a statement` will include the line items instead, where we can check all types there perhaps?